<testsuite failures="1" name="claroApp/portability/portability.feature" skipped="0" tests="1" time="1.470108">
  <testcase classname="claroApp.portability.portability" name="Validar erro ao enviar cityId inválido" time="1.470108">
    <failure message="portability.feature:346 - path: $.description, actual: 'cityId: Parâmetro fora do padrão!', expected: 'Required Long parameter 'cityId' is not present', reason: not equal">* header X-Application-Key = gwappkey ..................................... passed
* url portabilityUrl ...................................................... passed
* def responseApi = ....................................................... passed
Given path 'mobile-operators' ............................................. passed
And param cityId = '1231243434343324' ..................................... passed
When method get ........................................................... passed
Then status 400 ........................................................... passed
And match response == responseApi ......................................... failed

Stack Trace:
com.intuit.karate.exception.KarateException: portability.feature:346 - path: $.description, actual: 'cityId: Parâmetro fora do padrão!', expected: 'Required Long parameter 'cityId' is not present', reason: not equal
	at ✽.And match response == responseApi (portability.feature:346)</failure>
  </testcase>
</testsuite>
